controllers/acquire_cone.py

```diff
@@ -26,9 +25,6 @@ class AcquireConeController(StateMachine):
 
     EXTEND_VOLTAGE = tunable(3)
 
-    # swap the the side of the substation to test on a half field
-    swap_substation = tunable(False)
```

Comment:
Do we have a plan for testing cone pickup to score cycles in our limited space without this?

Reply:
Does changing the alliance on the DS whilst enabled work?

Reply:
I agree with Oliver - it will be much better to have a tunable that flips them. @whert-dev can you please make this change?